content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst

```diff
@@ -8,10 +8,10 @@ Terminology
 Warehouse
 ---------
 
-In Odoo, a **Warehouse** is the actual building/place in which a company's items are stocked.
-Multiple warehouses can be set up in Odoo and the user can create moves between warehouses.
+In Odoo, a *warehouse* is the actual building/place in which a company's items are stocked. Multiple
+warehouses can be set up in Odoo and the user can create moves between warehouses.
 
-.. _inventory/management/difference-warehouse-location:
+.. _inventory/warehouses_storage/inventory_management/warehouses-locations/locations:
```

Comment:
Hi @hojo-odoo, I'm not sure why this custom anchor was changed? 
The existing anchor `inventory/management/difference-warehouse-location` path is still valid and follows agreed upon format of `app-module/parent-folder(optional)/anchor-text`. The change you are making does not follow this format. Is there a reason this change is necessary, or can we leave the anchor as-is? LMK Thanks! 😊
```suggestion
.. _inventory/management/difference-warehouse-location:
```